client/my-sites/stats/activity-log/index.jsx

```diff
@@ -302,15 +304,47 @@ class ActivityLog extends Component {
 		);
 	}
 
+	renderNoLogsContent() {
+		const { filter, logLoadingState, siteId, translate } = this.props;
+
+		const isFilterEmpty = isEqual( emptyFilter, filter );
```

Comment:
There's a bit neater way of doing this here:

https://github.com/Automattic/wp-calypso/pull/25844/files#diff-50799a98f567c1cbecf662e97c90cbc9R38

```js
hasTokens() {
  return ! isEmpty( omit( this.props.filter, 'page' ) );
}
```

Since this might be needed in several places, could be good to have it in one shared place.

Reply:
I think i like your way better!

I'll update my PR

Reply:
I like how we don't need to `import` anything using `isEmpty`. ;-)